### ISSUE TITLE
Pass base URL as CLI argument

### ISSUE THREAD

**Issue by mbrodala** [Brunty/cigar#16]
Similar to https://github.com/symm/vape it should be possible to pass the base URL for smoke tests as CLI argument instead of hardcoding it in the `.cigar.json`.

This is essential for running the same batch of checks against multiple environments of the same project, e.g. staging and production.

Currently one can work around this with multiple files named after the environments and passing them to the CLI accordingly but this can quickly become a hassle to manage, especially if the files are not properly kept in sync.

**Comment by Brunty** [comment]
Good idea!

**Comment by Brunty** [comment]
@mbrodala version 1.6 has this :) https://github.com/Brunty/cigar/releases/tag/1.6

**Comment by mbrodala** [comment]
You are awesome. :-) This comes at the perfect time since this was preventing me from using cigar and just now I had a case where a smoke test would have prevented an error going live.

tl;dr: 👍

**Comment by Brunty** [comment]
@mbrodala you're more than welcome :)
